data/boarding missions.txt

```diff
@@ -24,6 +24,356 @@ mission "Assisting Merchant"
+				decline
+	destination Geminus
+
+mission "Assisting Navy (Atomics)"
+	assisting
+	repeat
+	to offer
+		random < 5
+	source
+		government Republic
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Navy's cause, the captain gives you <payment> and a message stating, "Buy Deep Sky Atomics today!."`
+				decline
+	destination Geminus
+
+mission "Assisting Navy (Atomics)"
+	assisting
+	repeat
+	to offer
+		random < 5
+	source
+		government Republic
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Navy's cause, the captain gives you <payment> and a message stating, "Buy Deep Sky Atomics today!."`
+				decline
+	destination Geminus
+
+mission "Assisting Navy (Join)"
+	assisting
+	repeat
+	to offer
+		random < 5
+	source
+		government Republic
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Navy's cause, the captain gives you <payment> and a message stating, "Join the Navy! See the Republic!"`
+				decline
+	destination Geminus
+	
+mission "Assisting Militia"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Militia
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Militia's cause, the captain gives you <payment>.`
+				decline
+	destination Girtab
+	
+mission "Assisting Militia (Complain)"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Militia
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Militia's cause, the captain gives you <payment> and a message saying, "You know we wouldn't get disabled this much if the Navy was here."`
+				decline
+	destination Girtab	
+	
+mission "Assisting Militia (Thank)"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Militia
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Militia's cause, the captain gives you <payment> and a message saying, "Thanks for the help friend!"`
+				decline
+	destination Girtab
+
+mission "Assisting Free Worlds"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government "Free Worlds"
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Free Worlds' cause, the captain gives you <payment>.`
+				decline
+	destination Bourne
+
+mission "Assisting Free Worlds (Safe)"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government "Free Worlds"
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Free Worlds' cause, the captain gives you <payment> along with a message saying, "Be Safe! Be Free!"`
+				decline
+	destination Bourne
+
+mission "Assisting Free Worlds (Join)"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
```

Comment:
Maybe add a check that ensures this won't trigger if the player has actually joined the Free Worlds.